### PR TITLE
Bigupdates

### PR DESCRIPTION
**Pull request by marklise**

### CODE REVIEW

File: force-app/main/default/layouts/Case-SA Request.layout-meta.xml [hunk 1]
```diff
@@ -126,10 +122,6 @@
                 <behavior>Edit</behavior>
                 <field>Provider_Validated__c</field>
             </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>Provider_Validated_Date__c</field>
-            </layoutItems>
```
Comment (janjulian) [inline force-app/main/default/layouts/Case-SA Request.layout-meta.xml:132]: @marklise did you intend to remove `Provider_Validated_Date__c` and `Patient_Validated_Date__c` from the layout?

File: force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html [hunk 2]
```diff
@@ -22,12 +22,12 @@ <h3 slot="title">
         </lightning-layout-item>
       </lightning-layout>
     </div>
-    <lightning-datatable
+    <c-poc-lightning-datatable
```
Comment (janjulian) [inline force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html:25]: Likely to break the build without `c-poc-lightning-datatable` in the codebase. You're on the right track though!

**Review by janjulian — CHANGES_REQUESTED** [review]
Question re: removed datetime fields on case layout, and remove reference to `c-poc-lightning-datatable`

File: force-app/main/default/layouts/Case-SA Request.layout-meta.xml (hunk 1 above)
Comment (marklise) [reply force-app/main/default/layouts/Case-SA Request.layout-meta.xml:132]: No - I'll update that

File: force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html (hunk 2 above)
Comment (marklise) [reply force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html:25]: Good catch! I'll fix.

File: force-app/main/default/layouts/Case-SA Request.layout-meta.xml (hunk 1 above)
Comment (marklise) [reply force-app/main/default/layouts/Case-SA Request.layout-meta.xml:132]: Fixed in 223b066.

File: force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html (hunk 2 above)
Comment (marklise) [reply force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.html]: Fixed in 223b066.

File: force-app/main/default/layouts/Case-SA Request.layout-meta.xml [hunk 3]
```diff
@@ -112,10 +112,6 @@
                 <behavior>Edit</behavior>
                 <field>Patient_Validated__c</field>
             </layoutItems>
-            <layoutItems>
-                <behavior>Edit</behavior>
-                <field>Patient_Validated_Date__c</field>
-            </layoutItems>
```
Comment (janjulian) [inline force-app/main/default/layouts/Case-SA Request.layout-meta.xml:118]: @marklise could you bring this one back as well?

**Review by janjulian — CHANGES_REQUESTED** [review]
One more page layout fix!

File: force-app/main/default/layouts/Case-SA Request.layout-meta.xml (hunk 3 above)
Comment (marklise) [reply force-app/main/default/layouts/Case-SA Request.layout-meta.xml]: Fixed in 38159b3.

**Review by janjulian — APPROVED** [review]

**Pull request merged**